dozer-ingestion/oracle/src/connector/replicate/log_miner/mod.rs

```diff
@@ -132,7 +132,9 @@ fn parse_and_map(
         return Ok(None);
     };
     Ok(Some(match parsed.kind {
-        ParsedLogManagerContentKind::Commit => MappedLogManagerContent::Commit(parsed.scn),
+        ParsedLogManagerContentKind::Commit => MappedLogManagerContent::Commit(
+            parsed.scn.ok_or(Error::NullValue(String::from("SCN")))?,
```

Comment:
NullValue is not created for this. We should "expect" in the parser once we know the op code is DML